src/osgPlugins/fbx/WriterNodeVisitor.cpp

```diff
@@ -444,7 +444,7 @@ WriterNodeVisitor::setLayerTextureAndMaterial(FbxMesh* mesh)
 }
 
 void
-WriterNodeVisitor::setControlPointAndNormalsAndUV(const GeometryList& geometryList,
```

Comment:
I am not seeing any change to this file in makOsg 3.4 so we should probably use the 3.6 version of it.

Reply:
Resolved.